.github/workflows/release-testflight.yml

```
@@ -0,0 +1,306 @@
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+      - name: Grant execute permission to scripts
+        run: |
+          chmod +x .github/scripts/rn_version_manager.sh
+          chmod +x .github/scripts/changelog_manager.py
+
+      - name: Increment version and sync native files
+        id: versioning
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:36: Double quote to prevent globbing and word splitting

(shellcheck)